### ISSUE TITLE
Adding new password on Windows does not create git commit

### ISSUE THREAD

**Issue by wiktor-k** [IJHack/QtPass#277]
Hello,

I've been testing QtPass and so far it's great but there is one issue: when a new password is created the password file is not committed immidiately to git but just added to staging (index) like so:

```
:~/.password-store(master)$ git status
On branch master
Your branch is ahead of 'origin/master' by 2 commits.
  (use "git push" to publish your local commits)
Changes to be committed:
  (use "git reset HEAD <file>..." to unstage)

        new file:   test4.gpg
```

Other modifications (removing, editing passwords) result in correct git commits:

```
:~/.password-store(master)$ git log
commit 3a586a3c0a61b0460aee5ccfaa007b1db7e7f7d2
Author: ...
Date:   Tue Jan 17 13:10:25 2017 +0100

    Remove for test2 using QtPass.

commit 1f211b921304584c4e9a37ed0f7c09b5dd77ec69
Author: ...
Date:   Tue Jan 17 12:44:01 2017 +0100

    Edit for test2 using QtPass.
```

I've got all necessary checkboxes ticked (Use git) and correct git paths (because editing and removing works seamlessly).

Is this by design or is this a bug? If it is a bug is there a way for me to debug it further?

Thanks!

QtPass version 1.1.3 installed via chocolatey.

**Comment by wiktor-k** [comment]
Updating to 1.1.6 fixed the issue, sorry for the noise :(

**Comment by annejan** [comment]
This was indeed probably an issue . .
Thanks for testing on 1.1.6

I'll see if I can poke the chocolatey maintainers to bump up qtpass to 1.1.6

**Comment by wiktor-k** [comment]
Yep, now I see your older comments on https://chocolatey.org/packages/qtpass

The moderation there is kind of annoying, it's not the first time that an outdated version there stays for a long time...

**Comment by wiktor-k** [comment]
Unfortunately this bug started to appear several days ago. It's not 100% reproducible but at least 80% of password creations do not create git commit.

I didn't update QtPass nor GPG4Win (everything stayed the same last time I reported). For what it's worth I'm using YubiKey as an auth token with "touch to decrypt" policy (so decryption always takes several seconds).

**Comment by ryankenney** [comment]
Still seeing this in Windows with QtPass 1.1.6. IMO, this is a pretty pretty big bug in the basic function of the product.

**Comment by nogeenharrie** [comment]
Old 2017 issue. Please reopen if still an issue.
